Adapter/ImageMediaAdapter.php

```diff
@@ -109,9 +110,19 @@ private function handlePayload(ImagePayload $payload): void
         $media = $bridge->getMedia();
         $uploadedFile = $this->syliusImageDownloader->downloadImage($payload);
         $this->updateMedia($media, $uploadedFile, $payload->getLocale());
+        $this->removeTemporaryMediaFile($uploadedFile);
         $this->entityManager->persist($bridge);
     }
 
+    private function removeTemporaryMediaFile(UploadedFile $uploadedFile): void
```

Comment:
@wachterjohannes this is the only new function, the rest of the MR is symfony 6 compatibility and php-cs-fixes